helloworld-rs/src/main/java/org/jboss/as/quickstarts/helloworldrs/HelloWorld.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The WildFly Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.jboss.as.quickstarts.rshelloworld;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+/**
+ * A simple REST service which is able to say "Hello World!"
+ *
+ * @author Ashwin Mehendale
+ * @author emartins
+ */
+
+@Path("/")
+public class HelloWorld {
+
+    @GET
+    @Path("/HelloWorld")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String getHelloWorldJSON() {
```

Comment:
The method name should be changed as this isn't a JSON response.

Also should we add a `@PathParam("name")` and great with the name instead of world? e.g. 

```java
@GET
@Path("/hello/{name}")
@Produces(MediaType.TEXT_PLAIN)
public String hello(@PathParam("name") final String name) {
    return String.format("Hello %s!", name);
}
```

Not needed, but just shows a simple path parameter too.

Reply:
+1

Reply:
updated the method name.